elements/rh-spinner/rh-spinner.ts

```diff
@@ -59,6 +43,22 @@ export class RhSpinner extends BaseSpinner {
       <slot></slot>
     `;
   }
+
+  // START hack for removal of contextProvider. delete for version 2.0
+  /**
+   * @deprecated Use Color context instead. See https://ux.redhat.com/foundations/color/context/
```

Comment:
Can the URL be removed while it's still a WIP? Don't want to confuse or frustrate people when that page doesn't exist yet.

Reply:
Oh, well then, Lidless Garlic Tub Musk